WeakAuras/Types.lua

```diff
@@ -703,49 +705,29 @@ WeakAuras.actual_unit_types_cast = {
   member = L["Specific Unit"],
 }
 
-WeakAuras.actual_unit_types = {
-  player = L["Player"],
-  target = L["Target"],
-  focus = L["Focus"],
-  pet = L["Pet"]
-}
-
-WeakAuras.threat_unit_types = {
+Private.threat_unit_types = {
   target = L["Target"],
   focus = L["Focus"],
   member = L["Specific Unit"],
   none = L["At Least One Enemy"]
 }
 
-WeakAuras.unit_types_range_check = {
+Private.unit_types_range_check = {
   target = L["Target"],
   focus = L["Focus"],
   pet = L["Pet"],
   member = L["Specific Unit"]
 }
 
-WeakAuras.unit_threat_situation_types = {
+Private.unit_threat_situation_types = {
   [-1] = L["Not On Threat Table"],
   [0] = "|cFFB0B0B0"..L["Lower Than Tank"],
   [1] = "|cFFFFFF77"..L["Higher Than Tank"],
   [2] = "|cFFFF9900"..L["Tanking But Not Highest"],
   [3] = "|cFFFF0000"..L["Tanking And Highest"]
 }
 
-WeakAuras.class_types = {}
-WeakAuras.class_color_types = {} -- TODO: it should be removed together with Bufftrigger (unused)
-for classID = 1, 20 do -- 20 is for GetNumClasses() but that function doesn't exists on Classic
-  local classInfo = C_CreatureInfo.GetClassInfo(classID)
-  if classInfo then
-    if WeakAuras.IsClassic() then
-      WeakAuras.class_types[classInfo.classFile] = WrapTextInColorCode(classInfo.className, select(4, GetClassColor(classInfo.classFile)))
-      WeakAuras.class_color_types[classInfo.classFile] = select(4, GetClassColor(classInfo.classFile))
-    else
-      WeakAuras.class_types[classInfo.classFile] = C_ClassColor.GetClassColor(classInfo.classFile):WrapTextInColorCode(classInfo.className)
-      WeakAuras.class_color_types[classInfo.classFile] = C_ClassColor.GetClassColor(classInfo.classFile):GenerateHexColorMarkup()
-    end
-  end
-end
```

Comment:
Code for populating Private.class_types mistakenly removed

Reply:
after re-adding it, i still have an issue on classic, aura loaded for warrior only show as loaded in WeakAurasOptions, but they are still not shown after closing
```
!WA:2!1zvZVnssvCmMrdybsjbyK2zrGNiTJ2rIjtI3D2f2LrdUtSN4qIT32otMLnRSRURYURjTRQNQQozsoGe(uK4MrIJiKflS81YI5JZOCIJLSqCaexmcoGGl5FGLxvD3ozgTyPe76RxvVFVFVFVxUQlnyj8s4t3FSmcjoypkwfCfN(H8JcMQipvTwvot1IEc5fepTrVEsIkN44KF8FwycI5hWfn5uMYB9k1BxXDQ1mUCfsr5SCLIfHrVyGsfjFJ7CNJq95Rq535v)wRToo6K6dUZxtG8nBu(AUsfsOk41JYOYGcoWxQcJP(CwhFEixSvE4Zz2NufgYlKG7E(besuzzeXx5AUUGrssyp7RrF1HEiX6jhmxU8JXejsflqksWiJ3T(CBo2At7PM40OD7g74w7bB2(mRJK4PVDUzMnvYGfvdr9LIg72E7A1R4eYr4txuqz9f84OcJILKomYHerGRFiskVPNewlKi2RSRBTgUUdIdv0RPl29CZwjmFEmtreuCG9S2d11rWdjfS74eoJyTDnCGJeIdxtFJccuVE0NsKfCmoVzMXykmLpy8JlmZCoyHWWdy8JybIEjyCHHzgm5UcH3zyW4SjnxaenifgAWN2hhrM6fRuCwdWDcrhp1oDa1)agrk)mZKd4CvqtbVVag3nAbWqLXpowQiycEhklySG0hUxJPeia7HiY054OHtTMWmmwqMuj8WDRv0jeXoW5eoFqohuSI3DSI7d3UeSYN(RElB0VuA0pyI5982PCYe27fb1lh9Euo3lyqJnNABcRVk4lo2hES8bTbR(j9aIwpA)cUOWOauElhPLpkKKFQpNhIbGS1r0is3OVq2yZ52GktEnrlCHX2ncd8mxaEzQZY2DfCFsWS5PuwAK(6dvcA)(Gh(dVPi9N)lhfayEq6GkwkIzwZOFZHyd7faILwZgGrEsUWBhoM0vVLBt(reXmzSND3nfeGF42QzLT3E(KTInKMjDwVCR2DA1USB7rrMtzUT8dT8LAyGsfdjEErqaNiSegZ4UUm0aGVn1mX8t1fYQ8aRAdWBwz7Mv3D7raDoXpkCtD9FGUHUP(T0VP2DP10B5TjbfQc0VBxDRU6D1puVN(r63rVV2RWemv(4ygWupK4LGLrlLIPjgCBEFQF0l2ZUjo7Lv3QOGaShwr17S27wKWWrlISNp9agWjFPykEY20J24KT2PF)kKZPM8ngk8Hj8Qhm7s8wdRDeIrhyr6xt)zRond27y8yHKa8eSu3WHbPmNrqsslLWsNU6qZidwOJCgGOSQAMMdaqK(jxvlGV(Cp)mN7H8pal4r1yvfaPayrwcY6WVbdmaIAE1Q3Q2gvCpYimV53878JYD)p6J(3ZKkip84nsFBbNb(WCvTahJQzGinXzXXadW1MjkF810nMKMk2b(7qDJrWQECbMiMM8vNer9sWYtsNXO7K)S0bj5s5ar1zPZCivsHKGUzBNau9ZA9KyKGuC9qcILzzEsY6Pxl6gg)SJ9F94caU7e1rrhaKlbXNAE25hnFll)srjathzacNPDBFbjZMP67uE32nI(82jn6hMWOVTKMtzy45j3hOCITt(DNM4QzIiw9LogMaT3X6RowyQIrmz0o1BuVIUHaWkZESLmUuzPrPoctfD9vxD1InRM49TqmjiqE7DiyAmqNVKpKENFPpXZm7JsMnVvMSZraQTNafnCV0FmD(9yPzxpfvUuv4j1QxVIBNKky2fZcorVWhdKN8dxdYJVW4gP5BKInPVOCrFL)Frm8XG2a1p40fo3qk7axgd3zaK7vQ8gBKcAWcNzxnRjIK9EPQYrlmpIbvMmjuDNz3JkRKdGkMXISElMyeJslGearDZIGULpXK3yFaIYQhIcJjjxDgsm7z35YLUBY6hpNgyV2edh9Y1m6fqjuY(RJaMbRVdsS)U1U9fJUDltXMj2JL80VxYGbuHaYhhBhCbD50f0)y97b6a)e9pnN(9d0)SU6FEE9ViN(xgO)v6pq)R7Q)WLFLv1)MC6j6FR(3Dp9Vpq)h2kx(nV))9VB(8h)4TXvYSXeRBN4p6pS0Tx7UgB(CMBZ7)bFzZNA5YN3lGq7hOU1Yu8q3YpOsrNYU2AGBtbOFXz2YPoMiRruv)E2MbQkipjgAH54MFQvxP0DTzfLsB)kHAw6skz6dt1lCsf9IwWjKEYjibUyBOiPIgnov9ldOEgPz0tdgNAZS1F(EoGTKk1KkO2D08(nIwOQGEsX3kgHb(gQy72JSnxkPyIRnFzI1j3mbiIw8zAK161Fpx77B5xsoAEDC9FAAMi(83vox7zd8KKbGiS7ARS6kV(uGEd5tnIST7wi9YOymH5wVYdR4AlgyA1JHNlW1m)QRC3j9eqXxOmd8SZFEsAzvZugraVwR7wPsDBZqz5Q2Nzjq1e6l1ueGeK0ou7uk9BCbLUmg3Gj3Fpc6GYqEOCFJufA)0Dk3prhRt14WW9cOkYeUGcXslMmEZgU1(2nQ3U82JnvfPwp76301pG4FqvD9Rm8qKGA6pAku60hoxGT(V7Hg25YV6QlZJw(BCpHFaI1NaNu)p3cO4)vZN)2RV59)hFFZNJZpmcqxtlh6)IT9nhBioyjPOh(v657)1x6W)8J(Fp
```